dps-web-experience-sample/Views/Shared/_DocsNav.cshtml

```diff
@@ -124,9 +124,9 @@
 
                 <li id="tutorial-ionic"><a href="~/en-us/docs/tutorial-ionic/">Get started with Ionic</a></li>
 
-                <li id="O365_Files"><a href="~/en-us/docs/O365_Files/">O365 Files API</a></li>
+                <li id="O365_Files"><a href="~/en-us/docs/o365_Files/">O365 Files API</a></li>
 
-                <li id="O365_Ionic"><a href="~/en-us/docs/O365_Ionic/">O365 Outlook and Ionic</a></li>
+                <li id="O365_Ionic"><a href="~/en-us/docs/o365_Ionic/">O365 Outlook and Ionic</a></li>
```

Comment:
Please make the same changes for the small_nav section as well. Also you will have to update index.js file.